src/Filter/ChainableFilterInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Filter;
+
+/**
+ * @author Javier Spagnoletti <phansys@gmail.com>
+ */
+interface ChainableFilterInterface extends FilterInterface
```

Comment:
Since it's highly related to the search, I would have move the interface to the `src/Search` folder, to be consistent with https://github.com/sonata-project/SonataAdminBundle/pull/7275/files#diff-239419a3056ac9e98522c27c2b452c432eb6f26260590ca42dc9cb6471116bbaR21